addons/web/static/src/less/views.less

```diff
@@ -70,3 +70,11 @@
 .oe_view.oe_cannot_create .oe_view_nocontent_create {
     display: none;
 }
+
+// text color light green
```

Comment:
This is not very modular. There is bootstrap variable @brand-success with the same value